docker-hub/scim.md

```diff
@@ -46,22 +48,70 @@ Before making SCIM configuration changes in your IdP, navigate to [Docker Hub](h
 
 The synchronization of user data is now automated, and the members in your Docker organization will now be automatically provisioned, updated, and de-provisioned based on the access control managed through your identity provider, Okta.
 
-### Generate a full-sync
+#### Generate a full-sync
 
-You must run full-sync after enabling SCIM, if you already have users assigned to the Docker Hub app when you enable SCIM. The full sync provisions the users that are assigned in the IdP Directory to  Docker Hub.
+You must run a full-sync after enabling SCIM, if you already have users assigned to the Docker Hub app. This provisions the users that are assigned in the IdP Directory to Docker Hub.
 
 1. Navigate to **Applications** > **Applications** and select the Docker Hub app.
 2. In the **Assignments** tab, click **Provision User** if you have pending users.
 3. Click **Apply to All** > **Reapply Mappings** and **Confirm**.
 
     > **Note**
     >
-    > Any user that was not previously provisioned is now provisioned in Docker Hub.
+    > This will generate a full-sync and any user that was not previously provisioned is now provisioned in Docker Hub.
+
+![scim-okta-button](images/generate-full-sync.png){:width="700px"}
+
+### Azure AD
+
+1. Navigate to Azure AD and select **AzureAD admin**.
+2. In the **Default Directory**, select **Add** > **Enterprise Application** > **Create your own application**.
+
+    > **Note**
+    >
+    > When you create an Enterprise Application you can configure SCIM with
+    > Security Assertion Markup Language (SAML).
+
+3. Type **Docker** for application name, select no-gallery, and click **Create**.
+4. In your Docker application, navigate to **Provisioning** and click **Get Started**.
+5. Select **Automatic** for the provisioning mode and enter your Admin credentials.
+
+    > **Note**
+    >
+    > You can access and copy your SCIM URL (Tenant URL) and API Token (Secret
+    > Token) in Docker Hub.
+
+6. Click **Test Connection** to enable and authorize the provisioning and click **Save**.
+7. In **Mappings**, click **Provision Azure Active Directory Groups** to disable Groups and click **Save**.
+8. In **Attribute Mappings**, keep the **userName**, **Active**, **givenName**, **familyName**, and delete the other attributes listed.
+
+    ![scim-okta-button](images/attribute-mapping.png){:width="700px"}
+
+9. Click **Start Provisioning** to begin the full synchronization.
+
+     > **Note**
+     >
+     > It can take up to 30 minutes to begin provisioning. You can also
+     > provision on demand provisioning with one user that is already assigned
+     > to the application.
+
+### Onelogin
```

Comment:
suggestion: Either remove **Onelogin** section because it just contains instructions on how to do full-sync, OR, complete with setup instructions for SCIM